### DISCUSSION TITLE
How to access OverlayStaticMesh and OverlaySkeletalMesh in C++?

### DISCUSSION THREAD

**Discussion by colorindarkness** [Sixze/ALS-Refactored#230] [Q&A] [answered]
I cannot find anywhere these are declared in the code, Yet in child Blueprints it looks like they are declared in the parent class. Any pointers would be great!
I'm working on some new ALSXT components and I would need to find socket locations for these meshes in C++ if possible.

**Comment by Sixze** [comment, marked as answer]
> Yet in child Blueprints it looks like they are declared in the parent class.

No, they are declared in `B_Als_Character` and are only used there, C++ or other bullprints do not interact with them in any way.

**Reply by colorindarkness** [reply]
Thank you. My workaround will be using BlueprintImplementable events.
